src/constants/colours.ts

```diff
@@ -19,3 +19,15 @@ export const BORDER_COLOURS = {
     orange: 'border-orange border-2',
     purple: 'border-purple border-2',
 } as const satisfies Record<Colour, string>;
+
+export const TECH_COLORS: { [key: string]: string } = {
+    'Beautiful Soup': '#F69800',
+    'Discord.py': '#7387CE',
+    'FastAPI': '#F6BF00',
```

Comment:
Replace `'FastAPI': '#F6BF00','` with `FastAPI: '#F6BF00',` to fix the prettier error.